### PR TITLE
Fix docs build of the examples in estimators rst

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
Fixes an issue with the build.

We also need a space between the title and descriptor such that when you hover over the tutorial it shows this!

### CODE REVIEW

File: docs/user-guide/estimators.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ arguments are translated into a TS difference assuming ``ts = n_sigma ** 2``.
 .. _sedtypes:
 
 SED types
-~~~~~~~~~
+^^^^^^^^^
```
Comment (morcuended) [inline docs/user-guide/estimators.rst:94]: The rest of section titles use `------`

Comment (Astro-Kirsty) [reply docs/user-guide/estimators.rst:94]: This is a subsection, as it belongs to the `Flux quantities` heading

Comment (morcuended) [reply docs/user-guide/estimators.rst:94]: Ah, right, I did not realized that

**Review by morcuended — APPROVED** [review]
Looks good to me

File: docs/user-guide/estimators.rst (hunk 1 above)
Comment (Astro-Kirsty) [reply docs/user-guide/estimators.rst:94]: No worries!

**Review by morcuended — DISMISSED** [review]
Thanks @Astro-Kirsty! It looks good to me.

**Comment by Astro-Kirsty** [comment]
Decided to update also other `rst` files that had some broken 'links' etc. I ran the docs and everything works nicely now

**Review by bkhelifi — APPROVED** [review]
Thanks a LOT for the effort!

**Review by cgalelli — APPROVED** [review]
Thanks a lot @Astro-Kirsty !

**Review by morcuended — APPROVED** [review]
Thanks @Astro-Kirsty, looks good!

**Pull request merged**